frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -90,16 +79,27 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 type ReduxProps = ConnectedProps<typeof connector>;
 
 type OwnProps = {
```

Comment:
Let me know what you think of the types, I'd like to do this more elegantly but I've stared at these types for too long so a fresh pair of eyes would be awesome for that